<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>640</width>
				<height>480</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Main Window</string>
		</property>
		<widget class="QWidget" name="Hueim">
			<layout class="QVBoxLayout" name="YO">
			<item>
			<widget class="QPushButton" name="pushButton">
				<property name="text">
					<string>PushButton</string>
				</property>
			</widget>
			</item>
			<item>
			<widget class="QTextEdit" name="textInput">
				<property name="text">
					<string>Input text to push</string>
				</property>
			</widget>
			</item>
			<item>
			<widget class="QTextEdit" name="textOutput">
				<property name="ReadOnly">
					<bool>true</bool>
				</property>
			</widget>
			</item>
			<item>
			<widget class="QPushButton" name="tristate">
				<property name="text">
					<string>Tristate Clown</string>
				</property>
			</widget>
			</item>
			</layout>
		</widget>
	</widget>

	<layoutdefault spacing="6" margin="11"/>
	<connections/>
</ui>